CrashReporter.xcodeproj/project.pbxproj

```diff
@@ -3500,6 +3504,8 @@
 		05CD33280EE9443A000FDE88 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				DEVELOPMENT_TEAM = 5Z97G9NZQ6;
+				GCC_WARN_ABOUT_DEPRECATED_FUNCTIONS = NO;
```

Comment:
Is it possible to ignore only in the few places that need it but enable globally?

Reply:
> Temporary disable deprecation warnings

It will be enabled back in #79